IDEAS/Buildings/Components/Interfaces/PartialZone.mo

```diff
@@ -3,6 +3,10 @@ partial model PartialZone "Partial model for thermal building zones"
   replaceable package Medium =
     Modelica.Media.Interfaces.PartialMedium "Medium in the component"
       annotation (choicesAllMatching = true);
+
+  parameter Modelica.Fluid.Types.Dynamics energyDynamicsAir=Modelica.Fluid.Types.Dynamics.FixedInitial
+    "Type of energy balance for air model: dynamic (3 initialization options) or steady state";
+  parameter Real mSenFac = 5 "Correction factor for thermal capacity of zone air.";
```

Comment:
So far it was kept on 5 (similar to TRNSYS) to account for furnitures, ... .
@rubenbaetens what's your opinion about this?